modules/shortcodes/js/jetpack-email-subscribe.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
```

Comment:
We generally don't include these comments in Jetpack. Only in WordPress.com's codebase.

Reply:
removed